todo-app/build/asset-manifest.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
So you ran `yarn build` or `npm run build`, causing CRA to build your application for a "production" deployment.

We want to ignore these build files in version control, as they will happen in individual environments depending on the deployment pipeline.

Unfortunately, it looks like the paths in the CRA `.gitignore` are not relative. So, because your `todo-app` project is nested inside another directory, the `.gitignore` rules aren't working correctly.

Can you please correct this, and not commit `build` files in the future? Let me know if you need any help changing the `.gitignore` rules to do this. Also you'll have to exclude previously tracked files via git.